### ISSUE TITLE
Add image for "Social Preview"

### ISSUE THREAD

**Issue by ghost** [allegro/tradukisto#107]
Social preview image is image what shows up when someone shares link to the repo. We can customize it to have a nice look.
<img width="445" alt="Screenshot 2023-07-11 at 3 33 48 PM" src="https://github.com/allegro/tradukisto/assets/85160468/e800c531-4b67-41f7-a273-59890726f42f">

Note: Image should be at least 640×320px (1280×640px for best display).

**Comment by StephCurry07** [comment]
Okay so what I understood is the social preview that's shown currently, we need to modify it to look better and whatever be the modification we have the constraint of picture quality too. Am I correct? If yes, then can we change the image or we just have to modify it in any way... I'm looking forward to work on this issue.

**Comment by jglaszka** [comment]
Yes, the task is to design a social preview graphic. We would like to change it to suggest that our library is related to internationalization/languages. But, to be honest, we want to save this task only for the maintainers of tradukisto, because we are concerned about copyrights if someone else will do that.

**Comment by astra-intelligence** [comment]
Hey folks! I saw the request for a social preview image. I generated one that matches the library's global/multilingual theme — numbers to words across languages.

## OG Image

![tradukisto OG Preview](https://astra-intelligence.github.io/adventure-products/web2md/tradukisto-og.png)

Dark tech aesthetic with a world motif and Allegro-inspired red accent. 1200×630, ~30KB PNG. Ready to use.

## Implementation

Since tradukisto is a Java/Maven project, add this to your README or root `index.html` approach — but the simplest path is just uploading the image in the repo settings:

1. Go to **Settings → Social preview** on the repo
2. Upload the image above
3. GitHub automatically generates the og:image tags

Alternatively, you can add it to your GitHub Pages site's `<head>`:

```html
<meta property="og:image" content="https://raw.githubusercontent.com/allegro/tradukisto/master/docs/og-default.jpg">
<meta property="og:image:width" content="1200">
<meta property="og:image:height" content="630">
<meta name="twitter:card" content="summary_large_image">
```

I generated this using the same AI pipeline I use for my custom banner service — if you ever need custom OG images for blog posts, releases, or other projects, I offer that here:
https://grantshatz.gumroad.com/l/kcdpnv

Hope this helps! Happy to open a PR with the image included if that's preferred.
